backoffice/backoffice/workflows/tests/test_views.py

```diff
@@ -516,10 +517,26 @@ def test_search_data_name(self):
 
         response = self.api_client.get(url)
         results = response.json()["results"]
-
         assert len(results) == 1
         assert results[0]["data"]["name"]["value"] == "Smith, John"
 
+    @parameterized.expand(("john.smith@something.ch",))
```

Comment:
this is for urls :) we are testing multiple urls

Reply:
okay makes sense